### PR TITLE
When forgetting the filesystem in Monitored mode, also forget Stratagem

### LINKED ISSUE #1094: When forgetting the filesystem in Monitored mode, also forget Stratagem
When the user clicks the forget action on a monitored fs:

Is Stratagem is configured for the fs, it should have it's `UnconfigureStratagemJob` run as part of the fs removal. 

An open question is what should happen if someone attempts to remove the fs while a scan is occuring, but this can probably be handled in another issue.

### PR DESCRIPTION
**Pull request by johnsonw**
Fixes #1094.

If Stratagem is configured for the fs, it should have it's UnconfigureStratagemJob run as part of the fs removal.

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: chroma_core/models/stratagem.py [hunk 1]
```diff
@@ -286,24 +337,16 @@ def get_body(mount_point, report_duration, purge_duration):
                         "rules": [
                             {
                                 "action": "LAT_COUNTER_INC",
-                                "expression": "&& < size 1048576 != type S_IFDIR",
-                                "argument": "SIZE < 1M",
-                            },
-                            {
-                                "action": "LAT_COUNTER_INC",
-                                "expression": "&& >= size 1048576000000 != type S_IFDIR",
+                                "expression": ">= size 1048576000000",
```
Comment (johnsonw) [inline chroma_core/models/stratagem.py]: I can put this into a separate patch if you guys would like. We were not picking up files that were >= 1tb. This change fixes that issue.

Comment (jgrund) [reply chroma_core/models/stratagem.py]: It's already landed in a separate patch.

Comment (johnsonw) [reply chroma_core/models/stratagem.py]: Ok cool, i'll remove this then.

**Comment by johnsonw** [comment]
Here are some screenshots:

Before forgetting the fs:
```
chroma=> select * from chroma_core_stratagemconfiguration;
 id |       state_modified_at       |   state    | immutable_state | filesystem_id |  interval  | report_duration | purge_duration | not_deleted 
----+-------------------------------+------------+-----------------+---------------+------------+-----------------+----------------+-------------
  8 | 2019-08-09 01:58:16.641666+00 | removed    | f               |             4 |  432000000 |                 |                | 
  1 | 2019-08-09 00:21:34.598848+00 | removed    | f               |             2 |  864000000 |                 |                | 
  9 | 2019-08-09 01:58:23.861189+00 | configured | f               |             4 |  432000000 |                 |                | t
  2 | 2019-08-09 00:22:34.422233+00 | removed    | f               |             2 |  864000000 |                 |                | 
  3 | 2019-08-09 00:55:11.462468+00 | removed    | f               |             3 |  864000000 |                 |                | 
  4 | 2019-08-09 01:10:26.040286+00 | removed    | f               |             3 |  864000000 |                 |                | 
  5 | 2019-08-09 01:11:22.275932+00 | removed    | f               |             3 | 2592000000 |                 |                | 
  6 | 2019-08-09 01:17:16.204285+00 | removed    | f               |             4 |  432000000 |                 |                | 
  7 | 2019-08-09 01:18:19.289055+00 | removed    | f               |             4 |  864000000 |                 |                | 
```

Confirm fs removal shows that stratagem will be unconfigured and set to removed.
image

image

After forgetting the filesystem:
```
chroma=> select * from chroma_core_stratagemconfiguration;
 id |       state_modified_at       |  state  | immutable_state | filesystem_id |  interval  | report_duration | purge_duration | not_deleted 
----+-------------------------------+---------+-----------------+---------------+------------+-----------------+----------------+-------------
  8 | 2019-08-09 01:58:16.641666+00 | removed | f               |             4 |  432000000 |                 |                | 
  1 | 2019-08-09 00:21:34.598848+00 | removed | f               |             2 |  864000000 |                 |                | 
  9 | 2019-08-09 01:59:54.388078+00 | removed | f               |             4 |  432000000 |                 |                | 
  2 | 2019-08-09 00:22:34.422233+00 | removed | f               |             2 |  864000000 |                 |                | 
  3 | 2019-08-09 00:55:11.462468+00 | removed | f               |             3 |  864000000 |                 |                | 
  4 | 2019-08-09 01:10:26.040286+00 | removed | f               |             3 |  864000000 |                 |                | 
  5 | 2019-08-09 01:11:22.275932+00 | removed | f               |             3 | 2592000000 |                 |                | 
  6 | 2019-08-09 01:17:16.204285+00 | removed | f               |             4 |  432000000 |                 |                | 
  7 | 2019-08-09 01:18:19.289055+00 | removed | f               |             4 |  864000000 |                 |                | 
```

From the cli:
```
[root@adm chroma-manager]# iml stratagem configure list
0 Stratagem configuration records found
```

I tested all of the cli functions and am happy to provide the output if you would like.

**Comment by jgrund** [comment]
I was asking if the Delete button on the stratagem config in the GUI still works.

**Comment by johnsonw** [comment]
For the delete functionality:

Configured:
image

After clicking delete:
image

**Pull request merged**